### PR TITLE
Fixed PR

### PR DESCRIPTION
**Pull request by dantovbein**

### CODE REVIEW

File: dist/mapd3.css [hunk 1]
```diff
@@ -1,3 +1,8 @@
+body {
```
Comment (rodrigocollavo) [inline dist/mapd3.css]: Im wondering if dist/ folder shouldn't be excluded from the project (.gitignore) as this is the output being generated by the build script.

File: src/charts/chart.js [hunk 2]
```diff
@@ -474,48 +403,53 @@ export default function Chart (_container) {
...
+    return (chartType === "line" || chartType === "area" || chartType === "stackedArea") ? Axis(cache.chart) : null;
+  }
+
+  function getLineChart () {
+    const { chartType } = config;
+    return (chartType === "line" || chartType === "area" || chartType === "stackedArea") ? Line(cache.panel) : null;
+  }
 
+  function csvToJson (csv) {
+    //let json = {"series": []};
```
Comment (rodrigocollavo) [inline src/charts/chart.js]: remove if not necessary

File: src/charts/data-manager.js [hunk 3]
```diff
@@ -201,6 +201,8 @@ export default function DataManager () {
     })
   }
 
+
```
Comment (rodrigocollavo) [inline src/charts/data-manager.js]: remove extra spaces

File: src/styles/charts/sunburst.scss [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+      z-index: -1;
+
+      .percentage {
+        font-size: 2.5em;
+      }
+    }
+  }
+}
+
+
```
Comment (rodrigocollavo) [inline src/styles/charts/sunburst.scss]: remove all white spaces below

File: src/charts/chart.js [hunk 5]
```diff
@@ -474,48 +403,53 @@ export default function Chart (_container) {
...
-    })
+    });
 
-    json["series"] = labels;
-    return json;
+    return { "series": [...labels] };
   }
 
-  function getObjectToPush(label, id, group, values) {
+  function getObjectToPush (label, id, group, values) {
```
Comment (rodrigocollavo) [inline src/charts/chart.js]: I see you formatted the code with a space between the function name and (, but the space was removed in lot of other places, not a problem, but probably you can reapply the auto-format used by the IDE, for this file and some others having the same issue.

**Review by rodrigocollavo — CHANGES_REQUESTED** [review]

File: dist/mapd3.css (hunk 1 above)
Comment (dtovbeinJC) [reply dist/mapd3.css]: Done

File: src/charts/chart.js (hunk 2 above)
Comment (dtovbeinJC) [reply src/charts/chart.js]: Removed

File: src/charts/data-manager.js (hunk 3 above)
Comment (dtovbeinJC) [reply src/charts/data-manager.js]: Removed

File: src/charts/chart.js (hunk 5 above)
Comment (dtovbeinJC) [reply src/charts/chart.js]: Formatted

File: src/styles/charts/sunburst.scss (hunk 4 above)
Comment (dtovbeinJC) [reply src/styles/charts/sunburst.scss]: Removed

**Comment by dtovbeinJC** [comment]
Rodrigo, all changes has been fixed. Please again make a review.

**Comment by dtovbeinJC** [comment]
Do not forget to build again the app

**Review by rodrigocollavo — APPROVED** [review]

**Pull request merged**